packages/common/src/services/logging/LogService.ts

```diff
@@ -0,0 +1,38 @@
+import { injectable, multiInject, optional } from 'inversify';
+
+import { LOG_TRANSPORTER } from '../../modules/types';
+
+import type LogTransporter from './transporters/LogTransporter';
+import { LogLevel } from './LogLevel';
+
+@injectable()
+export default class LogService {
+  constructor(@optional() @multiInject(LOG_TRANSPORTER) private readonly transporters: LogTransporter[]) {}
```

Comment:
Does this work without the symbol?
```suggestion
  constructor(@optional() @multiInject(LogTransporter) private readonly transporters: LogTransporter[]) {}
```

Reply:
See https://github.com/jwplayer/ott-web-app/pull/539#discussion_r1625828136